README.md

```diff
@@ -1,6 +1,6 @@
 # O3DE (Open 3D Engine)
 
-O3DE (Open 3D Engine) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
+O3DE (Open 3D Elephant) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
```

Comment:
```suggestion
O3DE (Open 3D Giraffe) is an open-source, real-time, multi-platform 3D engine that enables developers and content creators to build AAA games, cinema-quality 3D worlds, and high-fidelity simulations without any fees or commercial obligations.
```

Reply:
"Sign off and commit suggestion" nice!  
![image](https://user-images.githubusercontent.com/26804013/171258595-60988258-d5d5-44fd-b619-d7053222c4be.png)